test/e2e/parallel/registration_service_test.go

```diff
@@ -289,6 +290,26 @@ func TestSignupFails(t *testing.T) {
 		hostAwait := await.Host()
 		hostAwait.WithRetryOptions(wait.TimeoutOption(time.Second*15)).WaitAndVerifyThatUserSignupIsNotCreated(t, identity.ID.String())
 	})
+
+	t.Run("get signup for crtadmin with longer username fails", func(t *testing.T) {
```

Comment:
I think it would be useful to add a comment here to make it more clear that since our transformUsername code truncates the username this test is to make sure that creating a crtadmin with a longer username still fails

Reply:
@rajivnathan sure thing. added in [30b296f](https://github.com/codeready-toolchain/toolchain-e2e/pull/714/commits/30b296f08b20b8d8fab015998de9c85b467cf42d)